### DISCUSSION TITLE
Remove the concept of removing tags

### DISCUSSION THREAD

**Discussion by johnspurlock** [Podcastindex-org/podcast-namespace#592] [Ideas]
Not sure when this started coming up, but the idea of removing standard tags seems short-sighted and damaging to the entire effort.

There should be consideration prior to inclusion in the main namespace document - but once there, people should be able to rely on them being around indefinitely.

Whether or not current players pick them up right away is not a good standard, the only standard should be: _Is this a good idea in the long-term?_  Current players have their own short-term roadmaps, have nothing to do with the merit of a new tag.  And some current players have a difficult time with the most basic tags.

The spectre of possible deprecation removes any remaining incentive to get things right _prior_ to standardization, and disincentivizes possible implementors from implementing standardized tags, lest they be one of the unfortunately ones to be subsequently dropped.

**Comment by jamescridland** [comment]
I'm to blame here.

There's a tag that a) isn't flexible enough, b) isn't being used at all, c) is better served by an existing tag in Media RSS, which is being used.

Removing the tag - since it isn't used - would impact nobody.

Being confident and brave enough to realise when we fuck up is part of a mature plan. Apple has removed tags before. We shouldn't shy away from doing so.

**Reply by Dwev** [reply]
Agree! Removing the tag from the spec just moves it from being actively supported and promoted to being "depreciated" - the documentation and history is still there.  If we keep everything in the active spec, it just creates a whole heap of noise making it harder to get adoption.
